services/src/main/java/org/keycloak/keys/AbstractEcdsaKeyProviderFactory.java

```diff
@@ -43,7 +43,7 @@
             String.valueOf(GeneratedEcdsaKeyProviderFactory.DEFAULT_ECDSA_ELLIPTIC_CURVE),
             "P-256", "P-384", "P-521");
  
-    public final static ProviderConfigurationBuilder configurationBuilder() {
+    public static ProviderConfigurationBuilder configurationBuilder() {
```

Comment:
I think the final modifier might be here on purpose. It will prevent from hiding the method from subclass.

Reply:
Thanks for your comment @martin-kanis, I also thought about it, but it's static method. Is it make sense? I mean... It seems strange for me

Reply:
Of cource, I can revert it if you think it's necessary

Reply:
Let's see @hmlnarik view on this one.

Reply:
If "final" is removed it would make it possible to override from subclass, so this change should be reverted

Reply:
It was reverted